src/api/spec/controllers/source/key_info_controller_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Source::KeyInfoController, type: :controller do
+  describe 'GET #show' do
+    let(:user) { create(:confirmed_user) }
+    let(:project) { create(:project, name: "test_project", title: "Test Project") }
+    let(:backend_url) { CONFIG['source_url'] + Project::KeyInfo.send(:backend_url, project.name) }
+    let(:gpg_public_key) { Faker::Lorem.characters(1024) }
+    let(:ssl_certificate) { Faker::Lorem.characters(1024) }
+    let(:keyinfo_response) do
+      %(<keyinfo project="Test"><pubkey algo="rsa">#{gpg_public_key}</pubkey><sslcert>#{ssl_certificate}</sslcert></keyinfo>)
+    end
+
+    before do
+      Rails.cache.clear
+      stub_request(:get, backend_url).and_return(body: keyinfo_response)
```

Comment:
So basically from what I can see we have two options going forward on this point:
1. Continue stubbing the backend like its already done.
2. Setup obs signer by default in the vagrant machine and update this spec to create a project with a specified publickey key and ssl cert so that we can test that the rails app parse and returns the right values from the backend response.
I will definitely need some help if we are going to do #2 as I'm not too familiar with the vagrant setup scripts or how to create a project with a specific pubkey/ssl..